sched/test_ratio_predict.py

```diff
@@ -0,0 +1,265 @@
+"""
+Ratio scheduler is fast and fair scheduler based on weighted round-robin
+principle. Functional test for Ratio scheduler requires intensive loads to
+evaluate how fair the load distribution is.
+"""
```

Comment:
Just a copy & paste from `test_ratio_dynamic.py`? The file is dramatic copy from `test_ratio_dynamic.py` - in general it's OK for tests to copy & paste configuration and some other stuff, however I had a hard time to identify what's the difference between the files and had to use `vimdiff`.

Maybe the tests must be joined to the same test. Maybe it's good to introduce some common class for them. At least you should have been wrote some comment explaining what's the difference and why there is so massive copy & paste.